### PR TITLE
feat: open sourcing documensos design article

### PR DESCRIPTION
**Pull request by ElTimuro**

### CODE REVIEW

File: apps/marketing/content/blog/design-system.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  />
+
+  <figcaption className="text-center">
+   Documenso's Design System ✨
+  </figcaption>
+</figure>
+
+> TLDR; Our design system is OSS under MIT at [design.documenso.com](https://design.documenso.com)
+
+Today, we are open sourcing our design system, lovingly crafted by @Thilo. The system is public on Figma and will be available at design.documenso.com from here on out.
```
Comment (dephraiim) [inline apps/marketing/content/blog/design-system.mdx:29]: > @Thilo

Can you link the twitter handle or anything else? This doesn't link anywhere

Comment (dephraiim) [reply apps/marketing/content/blog/design-system.mdx:29]: Same for design.documenso.com, it doesn't link anywhere

File: apps/marketing/content/blog/design-system.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- Explorations of exciting aspects of signing and document handling:
+- What does signing look like when we no longer have skeuomorphic signatures?
+- What is signing if we move beyond paper-inspired documents?
+- What would the government process look like using Documenso?
+- Solutions Concepts for features further down the roadmap, e.g., Widgets in websites
+
+These are fascinating ideas for explorative design. They won't be built 1:1 but shape how we think about signing and where it can go, which is even more critical. If you are interested in product design, you are invited to join our [Discord](https://documen.so/discord) and discuss the future of signing design or the future of Documenso's design system. Also let me know what you think on [X (formerly Twitter)](https://x.com/eltimuro).
+
+Best from Hamburg\
```
Comment (dephraiim) [inline apps/marketing/content/blog/design-system.mdx:48]: Is the `\` intentional?

File: apps/marketing/content/blog/design-system.mdx (hunk 1 above)
Comment (ElTimuro) [reply apps/marketing/content/blog/design-system.mdx:29]: good points

Comment (ElTimuro) [reply apps/marketing/content/blog/design-system.mdx:29]: done

File: apps/marketing/content/blog/design-system.mdx (hunk 2 above)
Comment (ElTimuro) [reply apps/marketing/content/blog/design-system.mdx:48]: yes, it's a newline after Best from Hamburg

File: apps/marketing/content/blog/design-system.mdx (hunk 1 above)
Comment (Mythie) [reply apps/marketing/content/blog/design-system.mdx]: https://documen.so/design

Comment (Mythie) [reply apps/marketing/content/blog/design-system.mdx]: Right I see what you've done here

File: apps/marketing/content/blog/design-system.mdx (hunk 2 above)
Comment (dephraiim) [reply apps/marketing/content/blog/design-system.mdx:48]: Oh great

**Pull request closed without merging**